documentation/ntfs/ntfs_0.6.md

```diff
@@ -0,0 +1,705 @@
+equipment_id | chaine | Requis | Identifiant de l'équipement | 
+wheelchair_boarding | entier (1) | Optionnel | Accès UFR | 
+sheltered | entier (1) | Optionnel | Abris couvert
+elevator | entier (1) | Optionnel | Ascenseur
+escalator | entier (1) | Optionnel | Escalier mécanique
+bike_accepted | entier (1) | Optionnel | Embarquement vélo
+bike_depot | entier (1) | Optionnel | Parc vélo
+visual_announcement | entier (1) | Optionnel | Annonce visuelle
+audible_announcement | entier (1) | Optionnel | Annonce sonore
+appropriate_escort | entier (1) | Optionnel | Accompagnement à l'arrêt
+appropriate_signage | entier (1) | Optionnel | Information claire à l'arrêt
+
+    (1) Les valeurs possibles sont :
+        0 ou non spécifié - aucune information disponible
+        1 - l'équipement est disponible
+        2 - l'équipement n'est pas disponible
+
+### stops.txt (requis)
+Une ligne du fichier "stops.txt" représente un horaire d'arrivée et un horaire de départ à un point ou une zone.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+stop_id | chaine | Requis | Identifiant de l'arrêt
+visible | entier | Optionnel | Visible dans auto complétion ?
+stop_name | chaine | Requis | Nom de l'arrêt
+city_code | chaine | Optionnel | Identifiant de la commune (code INSEE). Si ce champ n'est pas renseigné, les coordonnées géographiques sont utilisées pour faire l'association.
+stop_lat | décimal | Requis | Latitude. Ce champ est obligatoire pour les arrêts physiques (location_type = 0) même si le champ geometry_id est renseigné afin de faciliter sa lecture. Il est conseillé pour les zones d'arrêts (location_type = 1), et inutile pour les autres cas.
+stop_lon | décimal | Requis | Longitude. Ce champ est obligatoire pour les arrêts physiques (location_type = 0) même si le champ geometry_id est renseigné afin de faciliter sa lecture. Il est conseillé pour les zones d'arrêts (location_type = 1), et inutile pour les autres cas.
+fare_zone_id | chaine | Optionnel | Zone tarifaire de l'arrêt 
+location_type | entier (1) | Requis | Type de l'arrêt ou de la zone
+geometry_id | géometrie | Optionnel | Ce champ est un lien vers le fichier geometries.txt qui décrit la géométrie associée à une zone géographique (type 2) afin de permettre au moteur de définir les adresses couvertes en cas de TAD zonal "adresse à adresse". Ce champ peut également être utilisé pour préciser une géométrie pour les zones d'arrêts (type 1) et les communes (type 4) pour enrichir le web service.
+parent_station | chaine | Optionnel | Identifiant de la zone d'arrêt, utilisé que sur des arrêts de type 0 (point d'arrêt)
+stop_timezone | timezones | Optionnel | Fuseau horaire, se référer à http://en.wikipedia.org/wiki/List_of_tz_zones
+equipment_id | chaine | Optionnel | Identifiant de la propriété accessibilité
+contributor_id | chaine | Requis | Identifiant du contributeur. Cette valeur peut être vide dans le cas de la déclaration d'une zone administrative (location_type=2)
```

Comment:
modified to location_type=3, location_type=2 has a possible link to a contributor